src/Common/test/Common.Expression.Test/Spring/TestResources/TestPerson.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+namespace Steeltoe.Common.Expression.Spring.TestResources
+{
+    public class TestPerson
+    {
+        private string name;
+        private TestAddress address;
+
+        public string Name
+        {
+            get => name;
+            set => name = value;
+        }
+
+        public TestAddress Address
+        {
+            get => address;
+            set => address = value;
+        }
```

Comment:
```suggestion
        public string Name { get; set; }

        public TestAddress Address { get; set; }
```

Reply:
This change will likely break a bunch of unit tests due to the way they test reading and writing properties vs fields... so I'm going to leave alone for now